.github/workflows/databricks.yml

```diff
@@ -27,17 +34,13 @@ jobs:
         uses: 'google-github-actions/auth@v0'
         with:
           credentials_json: ${{ secrets.CARTODB_ON_GCP_TERRAFORM_CI_CD_SERVICE_ACCOUNT_KEY }}
-
       - id: 'secrets'
         uses: 'google-github-actions/get-secretmanager-secrets@v0'
         with:
           secrets: |-
             databricks-ci-token:projects/cartodb-on-gcp-terraform-ci-cd/secrets/terraform-databricks-aws-ciuser-token
       - name: Check diff
         uses: technote-space/get-diff-action@v4
-        with:
```

Comment:
why is this removed?

Reply:
We are not using the filter of get-diff in the other clouds. Just a clean up to align scripts